core/integration/features/auctions/cancel-gfn-auction.feature

```diff
@@ -0,0 +1,104 @@
+
+    Then the network moves ahead "59" blocks
+    And the trading mode should be "TRADING_MODE_OPENING_AUCTION" for the market "ETH/DEC20"
+
+    When the parties place the following orders:
+      | party | market id | side | volume | price | resulting trades | type       | tif     |
+      | aux2  | ETH/DEC20 | buy  | 10     | 1000  | 0                | TYPE_LIMIT | TIF_GTC |
+      | aux   | ETH/DEC20 | sell | 10     | 1000  | 0                | TYPE_LIMIT | TIF_GTC |
+
+    Then the opening auction period ends for market "ETH/DEC20"
+
+    And the market data for the market "ETH/DEC20" should be:
+      | mark price | trading mode            | horizon | min bound | max bound | target stake | supplied stake | open interest |
+      | 1000       | TRADING_MODE_CONTINUOUS | 60      | 995       | 1005      | 7434         | 90000000       | 10            |
+      | 1000       | TRADING_MODE_CONTINUOUS | 600     | 978       | 1022      | 7434         | 90000000       | 10            |
+
+    # place GFN orders outside of bounds so we can trigger auction without those order uncrossing
+    When the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     |
+      | party1 | ETH/DEC20 | buy  | 1      | 990   | 0                | TYPE_LIMIT | TIF_GFN |
+      | party2 | ETH/DEC20 | sell | 1      | 1010  | 0                | TYPE_LIMIT | TIF_GFN |
+      | party4 | ETH/DEC20 | sell | 1      | 1010  | 0                | TYPE_LIMIT | TIF_GTC |
+    Then the order book should have the following volumes for market "ETH/DEC20":
+      | side | volume | price   |
+      | buy  | 1      | 1       |
+      | buy  | 2      | 900     |
+      | buy  | 1      | 990     |
+      | buy  | 1      | 1000    |
+      | sell | 2      | 1010    |
+      | sell | 2      | 1110    |
+      | sell | 1      | 200000  |
+      | sell | 1      | 1000000 |
+    And the market data for the market "ETH/DEC20" should be:
+      | mark price | trading mode            | horizon | min bound | max bound | target stake | supplied stake | open interest |
+      | 1000       | TRADING_MODE_CONTINUOUS | 60      | 995       | 1005      | 7434         | 90000000       | 10            |
+      | 1000       | TRADING_MODE_CONTINUOUS | 600     | 978       | 1022      | 7434         | 90000000       | 10            |
+
+    When the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     |
+      | party3 | ETH/DEC20 | buy  | 1      | 1010  | 0                | TYPE_LIMIT | TIF_GTC |
+    Then the trading mode should be "TRADING_MODE_MONITORING_AUCTION" for the market "ETH/DEC20"
+    ## The GFN orders will should be removed from the book (buy at 990, sell at 1010)
+    ## the 2 GTC orders that triggered the price auction, however, remain
+    And the order book should have the following volumes for market "ETH/DEC20":
```

Comment:
This test doesn't match the system-test. 

The system test check's explicitly that the GFN order is cancelled, this integration test checks that _some_ order at price level at `1110` is no longer on the book.

For parity, you also need to check that it was the GFN order that is no longer on the book, and not the GTC one. And also that its no longer on the book because it was cancelled, and not for any other reason.

Reply:
Added references to the orders, and verified the order status after entering auction